doc/classes/ProjectSettings.xml

```diff
@@ -1177,6 +1177,16 @@
 		<member name="rendering/quality/shadows/filter_mode.mobile" type="int" setter="" getter="" default="0">
 			Lower-end override for [member rendering/quality/shadows/filter_mode] on mobile devices, due to performance concerns or driver support.
 		</member>
+		<member name="rendering/quality/skinning/software_skinning_fallback" type="bool" setter="" getter="" default="true">
+			Allows [MeshInstance] to perform skinning on the CPU when the hardware doesn't support the default GPU skinning process with GLES2.
```

Comment:
Is it only for GLES2 as described here or also GLES3 now?

Reply:
The fallback is GLES2 specific, but `force_software_skinning` is applied to both GLES2 and GLES3. I'll add that information to `force_software_skinning`.